common/src/main/resources/assets/wynntils/urls.json

```diff
@@ -98,86 +90,86 @@
-    "url": "https://raw.githubusercontent.com/Wynntils/WynntilsWebsite-API/master/item_guesses.json"
+    "url": "https://raw.githubusercontent.com/Wynntils/Static-Storage/main/Data-Storage/ingredient_skins.json"
   },
   {
     "id": "dataStaticMajorIds",
     "md5": "38bb5299f66016271f5bf3a1f49ec0ae",
-    "url": "https://raw.githubusercontent.com/Wynntils/WynntilsWebsite-API/master/major_ids.json"
+    "url": "https://raw.githubusercontent.com/Wynntils/Static-Storage/main/Data-Storage/major_ids.json"
   },
   {
     "id": "dataStaticMaps",
-    "md5": "5a89584163267d54a3dfc0b357d26ad5",
-    "url": "https://raw.githubusercontent.com/Wynntils/WynntilsWebsite-API/master/maps/maps.json"
+    "md5": "d6353eb6b960f653922a11408597533d",
+    "url": "https://raw.githubusercontent.com/Wynntils/Static-Storage/main/Reference/maps.json"
   },
   {
     "id": "dataStaticMaterialConversion",
     "md5": "ae4b6270b0541c26f730a1bf33f7b1f9",
-    "url": "https://raw.githubusercontent.com/Wynntils/WynntilsWebsite-API/master/material_conversions.json"
+    "url": "https://raw.githubusercontent.com/Wynntils/Static-Storage/main/Data-Storage/material_conversions.json"
   },
   {
     "id": "dataStaticPlaces",
     "md5": "c1907ba6d7c8851be2c6cf3062d17de5",
-    "url": "https://raw.githubusercontent.com/Wynntils/Reference/main/locations/places.json"
+    "url": "https://raw.githubusercontent.com/Wynntils/Static-Storage/main/Reference/places.json"
   },
   {
     "id": "dataStaticSeaskipperDestinations",
     "md5": "9316de06eb80ef9a26f8b604e9ac13bd",
-    "url": "https://raw.githubusercontent.com/Wynntils/WynntilsWebsite-API/master/seaskipper_destinations.json"
+    "url": "https://raw.githubusercontent.com/Wynntils/Static-Storage/main/Data-Storage/seaskipper_destinations.json"
   },
   {
     "id": "dataStaticServices",
     "md5": "20c6bad6dddac9fdf32b582764e266ef",
-    "url": "https://raw.githubusercontent.com/Wynntils/Reference/main/locations/services.json"
+    "url": "https://raw.githubusercontent.com/Wynntils/Static-Storage/main/Reference/services.json"
   },
   {
     "id": "dataStaticSplashes",
-    "md5": "53341c283d2cc3c0e721844064a24603",
-    "url": "https://raw.githubusercontent.com/Wynntils/WynntilsWebsite-API/master/splashes.json"
+    "md5": "262f3c73093a1f0979beedb8b25e9420",
+    "url": "https://raw.githubusercontent.com/Wynntils/Static-Storage/main/Data-Storage/splashes.json"
   },
   {
     "id": "dataStaticLootrunTasksNamed",
     "md5": "4eb2931b1dbea269574b8864c8712ba4",
-    "url": "https://raw.githubusercontent.com/Wynntils/WynntilsWebsite-API/master/lootrun_tasks_named.json"
+    "url": "https://raw.githubusercontent.com/Wynntils/Static-Storage/main/Data-Storage/lootrun_tasks_named.json"
+  },
+  {
+    "id": "dataStaticShinyStats",
+    "md5": "2dbc0fdde50512b64828d5642bb480b6",
+    "url": "https://raw.githubusercontent.com/Wynntils/Static-Storage/main/Data-Storage/shiny_stats.json"
+  },
+  {
+    "id": "dataStaticAbilities",
```

Comment:
I think we've used alphabetic sorting on `id` in the rest of the file. At least, the intention was to do it. Maybe it needs to be rechecked in other places as well.

Reply:
It's formatted in a random order in most places though :). I will fix it up.

Reply:
It should sort automatically now with spotless, but I had to give up 2 space indentation, as I can only make it do 4 spaces. I think we should still keep it though :)

Reply:
I like that you fix automatic tools to enforce sorting! However, I think this change needs be made in Static-Storage instead, since we will just grab urls.json from there if it differs.

Also, I wonder if you can have Spotless pipe it further down to gson, which is what is enforcing the 2 space indentation we have on all other json files. It would be a bit of a shame if we couldn't get that consistent.

Reply:
Pushed a workflow in the Static-Storage repository. It also fixed the indentation. (And no, spotless only seems to format by the first rule it finds.)